### PR TITLE
:lipstick: changed css-class of mucicon and provided optional override #104

### PR DESCRIPTION
**Pull request by langehm**
Issues #104

### CODE REVIEW

File: src/components/Icon/MucIcon.vue [hunk 1]
```diff
@@ -1,15 +1,19 @@
 <template>
   <svg
     aria-hidden="true"
-    class="m-button__icon"
+    :class="[additionalClasses, iconClass]"
```
Comment (FabianWilms) [inline src/components/Icon/MucIcon.vue:4]: Can we not simply use `class` as a property? Having two properties do the same thing seems kind of complicated. A user can set a string to `class` which is seperated by spaces - and thus I think we only need one property.

Comment (FabianWilms) [reply src/components/Icon/MucIcon.vue]: I don't think it needs to be an array. What I imagine would be:

```html
<muc-icon class="class-a class-b class-c" />
```

Comment (langehm) [reply src/components/Icon/MucIcon.vue]: Should be resolved - used `Fallthrough Attributes` now with computed fallback value

**Review by FabianWilms — COMMENTED** [review]
Amazing! I like this :D  

one really little and small thing: could you update the readme, too?

https://github.com/it-at-m/muc-patternlab-vue/blob/029cd01df4024d6694f9840671ec67e7198720cf/README.md?plain=1#L74

**Comment by langehm** [comment]
> Amazing! I like this :D
> 
> one really little and small thing: could you update the readme, too?
> 
> https://github.com/it-at-m/muc-patternlab-vue/blob/029cd01df4024d6694f9840671ec67e7198720cf/README.md?plain=1#L74

I hesitated ... but here you go 🧐

**Review by FabianWilms — APPROVED** [review]

**Pull request merged**